client/src/components/ConfigureWebhook/index.js

```diff
@@ -30,6 +30,7 @@ export default function ConfigureWebhook(props) {
     if (props.webhookId) {
       setSelectedEvents(props.webhookSubscriptions);
     }
+    // eslint-disable-next-line
```

Comment:
Instead of disabling lint we should add `props.webhookSubscriptions` to the useEffect dependency array here.